organization/_posts/2021/2021-01-07-coordination.md

```diff
@@ -0,0 +1,143 @@
+
+Apologies: Liz Sexton-Kennedy, Josh McFayden
+
+## Review of 2020 and Planning for 2021
+
+See slides attached to the [agenda](https://indico.cern.ch/event/981562/), presented by Graeme Stewart.
+
+### Feedback on the presentation / Discussion Points
+
+#### Working groups
+
+- Caterina Doglioni: Should we build better bridges between HSF and experiments, taking the opportunity of the new batch of conveners?
+- Eduardo Rodrigues: in LHCb, we had a number of links / presentations. We keep the experimental committees aware of what we're doing and it's been working well.
+- Caterina Doglioni: LHCb seems covered, I was suggesting a formal/informal "introduction to the new HSF WG conveners" e.g. in weekly meetings of ATLAS / CMS.
+- Graeme: also worth looking back at mandates to check whether they are clear enough - we should make clear that we're here to be helpful. **ACTION**
+- Marc Paterno: we could write a letter to other experiments as well to let them know they're welcome to join.
+    - Graeme: we have good contacts with LHC and Belle-II, DUNE. We could write an email to the various experiments, pointing out the new conveners in 2021, and we can come and talk about HSF in the New Year. **ACTION**
+
+#### Workshops
+
+- Kevin Pedro: Zoom-fatigue problem -> make the workshops more focused on particular topics so that people pick and choose what to attend.
+- Graeme: the 4-day workshop could probably have been spread out in 4 different meetings... even if it was focused and had improved based on the May workshop feedback.
+
+---
+
+- Andi Salzburger: on workshop, "complaints" about length of workshops especially late in the year. More topical workshop: should the WGs be doing their individual "workshops"?
+- CD: agree with Andi. If no workshop, will we miss joint workshops with WLCG?
+    - Liz and Graeme are in the management board - need to also discuss there.
+- Andrea Valassi: are we actually managing to get people to talk together even when they go to the same workshop? There is the risk that people only attend their session.
+- Andi: if this is "out of the box" of the WGs, we could have joint sessions between WG A and WG B. This could be a working mode. 
+- Graeme: in particular we could also talk about I/O matters, which links analysis, frameworks and WLCG.
+- Conclusion from this part: explore more restricted but cross-cutting topics. No appetite for a spring workshop, but keep open option for autumn (TBD with WLCG). **ACTION**
+
+#### Training
+
+- Sudhir Malik - Training WG: 
+    - We (training) come from different experiments, so it would be good to understand what we are providing / what are our contributions in terms of simulation etc. Highlighting our success in terms of practical things. Some answer to "What have we achieved". **ACTION**
```

Comment:
```suggestion
    - We (training) come from different experiments, so it would be good to understand what we are providing / what are our contributions in terms of simulation etc. Highlighting our success in terms of practical things. Some answers to "What have we achieved". **ACTION**
```